test/dynamo/test_generator.py

```diff
@@ -640,13 +639,46 @@ def fn(t):
         self.assertEqual(y, t + sum(range(6)))
 
 
+class TestGeneratorSend(GeneratorTestsBase):
+    def test_send(self):
+        def double():
+            x = yield
+            yield x * 2
+
+        @torch.compile(backend="eager", fullgraph=True)
+        def fn(t):
+            gen = double()
+            next(gen)
+            return gen.send(t)
+
+        t = torch.randn(2)
+        y = fn(t)
+        self.assertEqual(y, t * 2)
+
+    def test_send_stop_iteration(self):
+        def double():
+            x = yield
+            yield x * 2
+
+        @torch.compile(backend="eager", fullgraph=True)
+        def fn(t):
+            gen = double()
+            next(gen)
+            a = gen.send(t)
+            b = gen.send(t)  # should result in StopIteration
+            return a + b
+
+        t = torch.randn(2)
+        with self.assertRaises(Unsupported):
+            fn(t)
```

Comment:
can you do a self.assertRaisesRegex? I want to know what the message looks like

Reply:
Dynamo will swallow the exception and raise `Unsupported("Observed exception")`